### DISCUSSION TITLE
Broken/Missing features after updaet

### DISCUSSION THREAD

**Discussion by alyter** [open-webui/open-webui#7186] [Issues]
# Bug Report

## Important Notes

- **Before submitting a bug report**: Please check the Issues or Discussions section to see if a similar issue or feature request has already been posted. It's likely we're already tracking it! If you’re unsure, start a discussion post first. This will help us efficiently focus on improving the project.

- **Collaborate respectfully**: We value a constructive attitude, so please be mindful of your communication. If negativity is part of your approach, our capacity to engage may be limited. We’re here to help if you’re open to learning and communicating positively. Remember, Open WebUI is a volunteer-driven project managed by a single maintainer and supported by contributors who also have full-time jobs. We appreciate your time and ask that you respect ours.

- **Contributing**: If you encounter an issue, we highly encourage you to submit a pull request or fork the project. We actively work to prevent contributor burnout to maintain the quality and continuity of Open WebUI.

- **Bug reproducibility**: If a bug cannot be reproduced with a `:main` or `:dev` Docker setup, or a pip install with Python 3.11, it may require additional help from the community. In such cases, we will move it to the "issues" Discussions section due to our limited resources. We encourage the community to assist with these issues. Remember, it’s not that the issue doesn’t exist; we need your help!

Note: Please remove the notes above when submitting your post. Thank you for your understanding and support!

---

## Installation Method

Git Clone

## Environment

- **Open WebUI Version:** 3.4 (About screen says 3.35 still)
- **Ollama (if applicable):** 3.12

- **Operating System:** Ubuntu 24.04.1
- **Browser (if applicable):** Firefox

**Confirmation:**

- [ x] I have read and followed all the instructions provided in the README.md.
- [ x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [ x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

Updates and new features to be in place.

## Actual Behavior:

After updating by git pull origin main, the about screen still shows v0.3.35 but says it's fully updated. The functions area is still in the workspace tab. I see no options for user groups. When navigating to connections in admin settings I just get a spinning wheel

## Description

**Bug Summary:**
[Provide a brief but clear summary of the bug]

## Reproduction Details

**Steps to Reproduce:**
Upgraded with git pull origin main. Was on 0.3.35 when attempting to update to current version.

## Logs and Screenshots
clovelace@Elroy:~$ sudo journalctl -u open-webui -f
Nov 21 13:37:21 Elroy bash[272569]: INFO:     127.0.0.1:43606 - "GET /_app/immutable/nodes/11.Dxst6_eU.js HTTP/1.1" 200 OK
Nov 21 13:37:32 Elroy bash[272569]: INFO:     127.0.0.1:58926 - "GET /ollama/api/version HTTP/1.1" 200 OK
Nov 21 13:37:32 Elroy bash[272569]: INFO:     127.0.0.1:58926 - "GET /api/version/updates HTTP/1.1" 200 OK
Nov 21 13:37:42 Elroy bash[272569]: INFO:     127.0.0.1:58932 - "GET /api/v1/users/ HTTP/1.1" 200 OK
Nov 21 13:40:23 Elroy bash[272569]: INFO:     127.0.0.1:44410 - "GET /ollama/api/version HTTP/1.1" 200 OK
Nov 21 13:40:23 Elroy bash[272569]: INFO:     127.0.0.1:44410 - "GET /api/version/updates HTTP/1.1" 200 OK
Nov 21 13:48:17 Elroy bash[272569]: INFO:     127.0.0.1:43660 - "GET /api/version/updates HTTP/1.1" 200 OK
Nov 21 13:48:26 Elroy bash[272569]: INFO:     127.0.0.1:47976 - "GET /api/version/updates HTTP/1.1" 200 OK
Nov 21 13:48:27 Elroy bash[272569]: INFO:     127.0.0.1:47976 - "GET /api/version/updates HTTP/1.1" 200 OK
Nov 21 13:48:28 Elroy bash[272569]: INFO:     127.0.0.1:47976 - "GET /api/version/updates HTTP/1.1" 200 OK
Nov 21 13:57:01 Elroy bash[272569]: INFO:     127.0.0.1:41764 - "GET /api/v1/auths/admin/config HTTP/1.1" 200 OK
Nov 21 13:57:01 Elroy bash[272569]: INFO:     127.0.0.1:41766 - "GET /api/webhook HTTP/1.1" 200 OK
Nov 21 13:57:03 Elroy bash[272569]: INFO:     127.0.0.1:41764 - "GET /ollama/urls HTTP/1.1" 404 Not Found
Nov 21 13:57:03 Elroy bash[272569]: ERROR [open_webui.apps.openai.main] 404, message='Not Found', url='https://api.openai.com/v1/urls'
Nov 21 13:57:03 Elroy bash[272569]: Traceback (most recent call last):
Nov 21 13:57:03 Elroy bash[272569]:   File "/home/clovelace/Elroy/open-webui/backend/open_webui/apps/openai/main.py", line 682, in proxy
Nov 21 13:57:03 Elroy bash[272569]:     r.raise_for_status()
Nov 21 13:57:03 Elroy bash[272569]:   File "/home/clovelace/Elroy/Orbit/lib/python3.12/site-packages/aiohttp/client_reqrep.py", line 1121, in raise_for_status
Nov 21 13:57:03 Elroy bash[272569]:     raise ClientResponseError(
Nov 21 13:57:03 Elroy bash[272569]: aiohttp.client_exceptions.ClientResponseError: 404, message='Not Found', url='https://api.openai.com/v1/urls'
Nov 21 13:57:03 Elroy bash[272569]: INFO:     127.0.0.1:41766 - "GET /openai/urls HTTP/1.1" 404 Not Found
Nov 21 13:57:03 Elroy bash[272569]: ERROR [open_webui.apps.openai.main] 404, message='Not Found', url='https://api.openai.com/v1/keys'
Nov 21 13:57:03 Elroy bash[272569]: Traceback (most recent call last):
Nov 21 13:57:03 Elroy bash[272569]:   File "/home/clovelace/Elroy/open-webui/backend/open_webui/apps/openai/main.py", line 682, in proxy
Nov 21 13:57:03 Elroy bash[272569]:     r.raise_for_status()
Nov 21 13:57:03 Elroy bash[272569]:   File "/home/clovelace/Elroy/Orbit/lib/python3.12/site-packages/aiohttp/client_reqrep.py", line 1121, in raise_for_status
Nov 21 13:57:03 Elroy bash[272569]:     raise ClientResponseError(
Nov 21 13:57:03 Elroy bash[272569]: aiohttp.client_exceptions.ClientResponseError: 404, message='Not Found', url='https://api.openai.com/v1/keys'
Nov 21 13:57:03 Elroy bash[272569]: INFO:     127.0.0.1:52162 - "GET /openai/keys HTTP/1.1" 404 Not Found

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by alyter** [comment]
Any insight on this one?

**Comment by gulaschnascher4000** [comment]
Same Problem.

**Comment by alyter** [comment]
Still having the same issue. The problem is definitely in going from 0.3.35 to a higher version. Any insight or solution?
